### ISSUE TITLE
'after' is indented too much on 'receive'

### ISSUE THREAD

**Issue by elbrujohalcon** [AdRoll/rebar3_format#183]
Currently, `default_formatter` formats this code…

```erlang
receive
    {'DOWN', Ref, _Type, Pid, _Reason} ->
        finished
after Timeout ->
        error
end.
```

…like this…

```erlang
    receive
        {'DOWN', Ref, _Type, Pid, _Reason} ->
            finished
        after Timeout ->
            error
    end.
```

I would like it to be formatted like this…

```erlang
    receive
        {'DOWN', Ref, _Type, Pid, _Reason} ->
            finished
    after Timeout ->
        error
    end.
```

**Comment by paulo-ferraz-oliveira** [comment]
Related to https://github.com/AdRoll/rebar3_format/issues/147.

**Comment by elbrujohalcon** [comment]
HA! It's not related, it's the same thing 🤦 
But I "fixed" #147… I actually only fixed it for `try…after`.
Shame on me! Anyway… Now we have another ticket to track the **actual** fix.
